txpool/txpool.proto

```diff
@@ -9,6 +9,18 @@ option go_package = "./txpool;txpool";
 
 message TxHashes { repeated types.H256 hashes = 1; }
 
+message Tx {
```

Comment:
Hi. We have some small naming convention:
`tx` it's database transaction
`txn` it's ethereum transaction (so use it here plz)
`txs` many `txn`'s